### PR TITLE
Weekly finishing touches

### PR DESCRIPTION
**Pull request by walaura**
## Why are you doing this?

- Clear up the react components, especially the hero and the radio buttons to make more reusable in the future (the aim is to extract them as needed for the future pages)
- Update the wording to the final copy
- Only preselect 6 for 6 offer with a specific query string
- Fix the buttons in IE11
- Fix the selected input contrast ratio

## Screenshots
support thegulocal com_uk_subscribe_weekly ipad

### CODE REVIEW

File: assets/pages/weekly-subscription-landing/components/weeklyCta.scss [hunk 1]
```diff
@@ -21,12 +23,13 @@
...
 .weekly-cta svg {
-  display: inline-block;
-  vertical-align: middle;
+  flex: 0 0 auto;
+  display: block;
 }
 
 .weekly-cta svg {
   height: $gu-v-spacing * 2;
+  width: 2em;
```
Comment (JustinPinner) [inline assets/pages/weekly-subscription-landing/components/weeklyCta.scss]: Don't we usually express in `px` and let the compiler convert to `em`? I may be wrong.

File: assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx [hunk 2]
```diff
@@ -97,23 +103,18 @@ const content = (
...
-          (&quot;GNM&quot;) reserves the right to withdraw this promotion
-          at any time. View full promotion terms and conditions here.
+          <p>Subscriptions available to people aged 18 and over with a valid email address. For full details of Guardian Weekly print subscription services and their terms and conditions - see <a href="https://www.theguardian.com/guardian-weekly-subscription-terms-conditions">here</a>
           </p>
         </WeeklyTextBlock>
         <WeeklyTextBlock title="Guardian Weekly terms and conditions">
-          <p>Offer subject to availability. Guardian News and Media Limited
-          (&quot;GNM&quot;) reserves the right to withdraw this promotion
-          at any time. View full promotion terms and conditions here.
+          <p>Offer subject to availability. Guardian News and Media Limited (&quot;GNM&quot;) reserves the right to withdraw this promotion at any time. For full promotion terms and conditions see <a href="https://subscribe.theguardian.com/p/WWM99X/terms?country=GB">here</a>.
```
Comment (JustinPinner) [inline assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx]: Beware `?country=GB`. The terms page on subscribe uses different footer content according to territory (for local contact details etc)

File: assets/pages/weekly-subscription-landing/components/weeklyHero.jsx [hunk 3]
```diff
@@ -2,50 +2,55 @@
...
-          sizes="(max-width: 740px) 90vw, 500px"
+          sizes="(max-width: 740px) 90vw, 600px"
           imgType="png"
         />
         <SvgWeeklyHeroCircles />
-        <HeadingBlock overheading="The Guardian Weekly subscriptions" heading="Seven days of international news curated to give you a clearer global perspective." />
+        <HeadingBlock overheading={overheading} heading={heading} />
       </LeftMarginSection>
     </div>
+    {cta &&
```
[the remark this answers is not in the document]
Comment (walaura) [reply assets/pages/weekly-subscription-landing/components/weeklyHero.jsx]: yep, Afaik it's a common react idiom?

File: assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx (hunk 2 above)
Comment (walaura) [reply assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx]: oh! good point. The country code should probably carry from this page's code

**Pull request merged**